demos/oauth2-browser/config.js.template

```diff
@@ -0,0 +1,10 @@
+/*
+You can generate your own clientid by creating an application on the ArcGIS for Developers site. Be sure to add http://localhost:8080/ as a redirect uri for your application
+
+once you have a clientid of your own, copy/paste it here and rename this file 'config.js'
+*/
+let config = {
+  clientId: "QVQNb3XfDzoboWS0",
```

Comment:
this wasn't an accident. there's nothing inherently dangerous about sharing a `clientId` because its the public component of a registered application and the only redirect_uri that it is configured to respect is `http://localhost:8080`.

this way folks don't _have to_ register an application of their own in order to take it for a test drive.